Makefile

```diff
@@ -5,7 +5,7 @@ PODMAN_VERSION ?= 3.4.4
 BUNDLE_EXTENSION = crcbundle
 CRC_VERSION = 2.1.0
 COMMIT_SHA=$(shell git rev-parse --short HEAD)
-MACOS_INSTALL_PATH = /Applications/CodeReady Containers.app/Contents/Resources
+MACOS_INSTALL_PATH = /Applications/Red Hat OpenShift Local.app/Contents/Resources
```

Comment:
also need to update the directory path in [L#316](https://github.com/code-ready/crc/blob/609ecb246881d17325048e2b21301f7a1522122d/Makefile#L316)

Reply:
different format in those lines due to the spaces.